mmv1/templates/terraform/examples/chronicle_watchlist_basic.tf.tmpl

```diff
@@ -0,0 +1,21 @@
+provider "google-beta" {
+ user_project_override = true
+}
```

Comment:
Are these lines needed to configure google-beta provider?

Reply:
I can try removing and test! Thanks for the suggestion.

Reply:
Removing them worked.